data/binding/gen.go

```diff
@@ -130,25 +132,26 @@ func {{ .Name }}ToStringWithFormat(v {{ .Name }}, format string) String {
 	return str
 }
 
-func (s *stringFrom{{ .Name }}) Get() string {
-	val := s.from.Get()
+func (s *stringFrom{{ .Name }}) Get() (string, error) {
+	val, err := s.from.Get()
 
-	return fmt.Sprintf(s.format, val)
+	return fmt.Sprintf(s.format, val), err
```

Comment:
Only format the value if err is nil, we don't want to be returning errors and non-empty results at the same time;
```go
val, err := s.from.Get()
if err != nil {
    return "", err
}
return fmt.Sprintf(s.format, val), nil
```